backend/database-fs.js

```diff
@@ -19,28 +18,15 @@ export function clear (): Promise<void> {
     .then(keys => Promise.all(keys.map(key => unlink(join(dataFolder, key)))))
 }
 
-export function exportToJSON (): Promise<Object> {
-  return readdir(dataFolder)
-    .then(keys => Promise.all(keys.map(key => readData(key).then(value => ([key, value])))))
-    .then(Object.fromEntries)
-}
-
-export function importFromJSON (json: Object): Promise<void> {
-  // TODO: rollback the operation if an error occurs?
-  return Promise.all(Object.keys(json).map(key => writeData(key, json[key])))
-    .then(() => undefined)
-}
-
 // eslint-disable-next-line require-await
-export async function readData (key: string): Promise<Buffer | void> {
+export async function readData (key: string): Promise<Buffer | string | void> {
+  // Necessary here to thwart path traversal attacks.
   checkKey(key)
   return readFile(join(dataFolder, key))
-    .then(buffer => key.startsWith('blob=') ? buffer : buffer.toString('utf8'))
     .catch(err => undefined) // eslint-disable-line node/handle-callback-err
 }
 
 // eslint-disable-next-line require-await
 export async function writeData (key: string, value: Buffer | string): Promise<void> {
-  checkKey(key)
```

Comment:
We do also want to make sure no path traversal attacks happen here (files outside of the data directory must not be written to)

Reply:
Sure, that's done in 'chelonia/db/set', This writeFile() function is not meant to be called directly (except in tests)